geomesa-process/geomesa-process-vector/src/main/scala/org/locationtech/geomesa/process/tube/TubeBuilder.scala

```diff
@@ -97,6 +97,55 @@ abstract class TubeBuilder(val tubeFeatures: SimpleFeatureCollection,
     }
   }
 
+  /**
+    * This function checks if a segment crosses the IDL.
+    * @param point1 The first point in the segment
+    * @param point2 The second point in the segment
+    * @return boolean true if the segment crosses the IDL, otherwise false
+    */
+  def crossesIDL(point1:Coordinate, point2:Coordinate): Boolean = {
```

Comment:
These are all generally useful utilities.  Please move them into the `geomesa-util` module and give them some specific tests.  Also, please see my email for a slight change to the computation.